engwales-plead.sty

```diff
@@ -25,3 +26,10 @@
   \begin{flushright}\uline{Defendant}\end{flushright}
   \endgroup
 }
+
+%% Allow for you to list items with the number in the margin
+\newcounter{point}
+\newenvironment{point}[1][]
+{\begin{enumerate}[leftmargin=4em,labelsep=1.5em]\refstepcounter{point}
```

Comment:
the enumitem package supports enumeration continuation. custom counter is unnecessary.

Reply:
The resume option doesn't carry across sections, I thought a custom one would make sure the counting didn't reset.

Reply:
i think it does?

MWE:

```tex
\documentclass{article}
\usepackage{enumitem}
\def\items{\item foo bar baz \item foo \item foo bar}
\begin{document}
\begin{enumerate}\items\end{enumerate}
\section{hello}
\begin{enumerate}[resume]\items\end{enumerate}
\section{world}
\begin{enumerate}[resume]\items\end{enumerate}
\end{document}
```

result:

![image](https://user-images.githubusercontent.com/10174018/107866234-6b6abc80-6e23-11eb-9958-f7ba98a46253.png)

Reply:
Just messed around with it, and the resume option doesn't seem to work from within the point env. So I'll see about creating a new list as suggested [here](https://github.com/passamo9/LaTeX_EnglandWales_Templates/pull/10#discussion_r575738939), but I might wait for passamo9 to weigh in below on the use of it.